### PR TITLE
Make storage drivers (for each tokens persistence) configurable

### PR DESCRIPTION
**Pull request by cermakjiri**

### CODE REVIEW

File: docs/usage-examples/customStorageDriver.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    async remove(key) {
+        localStorager.removeItem(key);
+    },
+};
+```
+
+### 2. Pass the custom driver to config
+
+```js
+import { configure, TokensPersistence, StorageDrivers } from '@ackee/petrus';
```
Comment (horaklukas) [inline docs/usage-examples/customStorageDriver.md]: Import of the `StorageDrivers` variable is useless, isn't it?

File: src/config/storageDrivers/indexedDB.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { openDB } from 'idb';
+
+import { globalEnv } from '../global';
+
+const noop = () => {};
```
Comment (horaklukas) [inline src/config/storageDrivers/indexedDB.js]: There is also `noop` fnc in lodash, you know right?

File: src/modules/tokens/modules/storage/sagas/storageHandlers.js [hunk 3]
```diff
@@ -1,51 +1,26 @@
...
-            break;
-
-        case SESSION:
-            yield storage.indexedDB.remove(TOKENS_KEY);
-            storage.session.setItem(TOKENS_KEY, JSON.stringify(tokens));
-            break;
-
-        default:
-    }
+    yield storageDriver(persistence || forcedPersistence).set(TOKENS_KEY, tokens);
```
Comment (horaklukas) [inline src/modules/tokens/modules/storage/sagas/storageHandlers.js]: I don't understand, why did you swapped them? Does it make sense to name variable `forcedPersistence` if it actually doesn't force the value and is more a fallback persistence? 🙂

File: docs/usage-examples/customStorageDriver.md (hunk 1 above)
Comment (cermakjiri) [reply docs/usage-examples/customStorageDriver.md]: Ye, it's. I've forgotten to remove it.

File: src/config/storageDrivers/indexedDB.js (hunk 2 above)
Comment (cermakjiri) [reply src/config/storageDrivers/indexedDB.js]: I know, but this is more bundle-size efficient.

File: src/modules/tokens/modules/storage/sagas/storageHandlers.js (hunk 3 above)
Comment (cermakjiri) [reply src/modules/tokens/modules/storage/sagas/storageHandlers.js]: Ye, that's error, it shouldn't be swapped :)

**Pull request merged**